src/AddTodoForm.jsx

```diff
@@ -1,12 +1,25 @@
-function AddTodoForm(props) {
+import { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  function handleTitleChange(event) {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
+
   function handleAddTodo(event) {
     event.preventDefault();
     console.log(event.target);
     console.log(event.target.title);
-    const todoTitle = event.target.title.value;
+    const newTodo = {
+      title: todoTitle,
+      id: Date.now(),
+    };
+
     console.log(todoTitle);
-    props.onAddTodo(todoTitle);
-    event.target.reset();
+    onAddTodo(newTodo);
+    setTodoTitle("");
   }
 
   return (
```

Comment:
Not related to your changes here, but I think we should remove the `<h1>AddTodoForm</h1>` below. It's very large and takes up a lot of space, and if I were a user of the app I wouldn't know what AddTodoForm means.